src/components/flyout/OverlayHeader.scss

```diff
@@ -0,0 +1,17 @@
+@import '../../styles/variables';
+
+.bdl-OverlayHeader {
+    position: relative;
+    z-index: 1;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    height: 64px;
```

Comment:
Ideally, we want to avoid setting dimensions and allow content to do it for us. If this is absolutely necessary, let's use `min-height` so we don't see content overflow issues.